source/quickinstallationguide/qig.rst

```diff
@@ -490,7 +490,7 @@ the system VMs images.
   
    /usr/share/cloudstack-common/scripts/storage/secondary/cloud-install-sys-tmplt \
    -m /export/secondary \
-   -u http://download.cloudstack.org/systemvm/4.11/systemvmtemplate-4.11.1-kvm.qcow2.bz2 \
+   -u http://download.cloudstack.org/systemvm/4.11/systemvmtemplate-4.11.2-kvm.qcow2.bz2 \
```

Comment:
q. are you wanting this guide to keep 'itself' up to date or keep manual control of guide changes?
if the former, there is a global variables file  ../source/_global.rst  which you can use (or even add to).

.. Latest version systemvm template name
.. |sysvm64-name-kvm|    replace:: systemvm-kvm-4.11.2

.. Latest version systemvm template URL
.. |sysvm64-url-kvm|    replace:: http://download.cloudstack.org/systemvm/4.11/systemvmtemplate-4.11.2-kvm.qcow2.bz2

so you can use: |sysvm64-url-kvm|

Reply:
That's a good point, but since we are referencing 4.11 branch (in repo file), unless there is a variable for this (and it doesn't make sense possibly as the time goes by), I would stick to manual URLs for this very doc ?